packages/manager/apps/web/client/app/domain/webhosting/domain-webhosting.html

```diff
@@ -0,0 +1 @@
+<div data-ui-view="webhostingView"></div>
```

Comment:
Is this necessary ? Looks to me you're adding a file for something that could be included in a template